CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.18.9] - 2021-01-21
```

Comment:
You have added new features. Hence it needs to be 0.19.0.

Reply:
Changelogs shouldn't be in a PR. They created from the PR-titles after the merge ;)

Reply:
But for 0.x-releases, it is ok to add backward-compatible changes in an 0.x.x-release, since 0.x-release could contain bc-breaks and therefore not automatically updated by composer and co.

Reply:
@koertho Ah I see, thanks for clearification.